website/docs/r/event_orchestration_integration.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: 'pagerduty'
+page_title: 'PagerDuty: pagerduty_event_orchestration_integration'
+sidebar_current: 'docs-pagerduty-resource-event-orchestration-integration'
+description: |-
+  Creates and manages an Integration for an Event Orchestration.
+---
+
+# pagerduty_event_orchestration_integration
+
+An Event Orchestration Integration allows you to create and manage multiple Integrations (and Routing Keys) per Event Orchestration _and_ will alow you to move (migrate) Integrations _between_ two Event Orchestrations.
+
+## Example of configuring an Integration for an Event Orchestration
+
+This example shows creating `Event Orchestration` and `Team` resources followed by creating an Event Orchestration Integration to handle Events sent to that Event Orchestration.
+
+-> When a new Event Orchestration is created there will be one Integration (and Routing Key) included by default. Example below shows how to create an extra Integration associated with this Event Orchestration.
+
+```hcl
+resource "pagerduty_team" "database_team" {
+  name = "Database Team"
+}
+
+resource "pagerduty_event_orchestration" "event_orchestration" {
+  name = "Example Orchestration"
+  team = pagerduty_team.database_team.id
+}
+
+resource "pagerduty_event_orchestration_integration" "integration" {
+  event_orchestration = pagerduty_event_orchestration.event_orchestration.id
+  label = "Example integration"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `event_orchestration` - (Required) ID of the Event Orchestration to which this Integration belongs to. If value is changed, current Integration is associated with a newly provided ID.
+- `label` - (Required) Name/description of the Integration.
+
+-> Modifying `event_orchestration` property will cause Integration migration process and as a result all future events sent with this Integrations's Routing Key will be evaluated against the new Event Orchestration.
```

Comment:
**Annotation:** Note for a user which indicates expected behavior on `event_orchestration` property change